server/overallReviews.js

```diff
@@ -2,7 +2,7 @@ const express = require('express');
 const router = express.Router();
 const Review = require('../database/connection.js');
 
-//Render overall Rating bars
+//Render overall Rating bars//LEGACY CODE
```

Comment:
I'm not sure if this is a good way to render a comment - but I think it's ultimately up to personal preference.